### DISCUSSION TITLE
Welcome to aiomusiccast Discussions!

### DISCUSSION THREAD

**Discussion by vigonotion** [vigonotion/aiomusiccast#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by plimptm** [comment]
Hi @vigonotion, I'm somewhat new to home assistant but have several Musiccast devices and have been looking to integrate them into a local network automation instance. I work in server/systems automation, including some Python experience, so the backend concepts are pretty comfortable to for me.

I'd very much like to contribute to this codebase in any ways that would be helpful. Right now I'm working on some scripts in my home assistant to handle input from genetic hardware buttons/controllers like the lutron caseta pico controllers.

Just saying hi for now and I'll start following this repo and any issues that might pop up.
